packages/starlight/index.ts

```diff
@@ -88,13 +88,15 @@ function vitePluginStarlightUserConfig(
   opts: StarlightConfig,
   { root }: AstroConfig
 ): NonNullable<ViteUserConfig['plugins']>[number] {
+  const resolveRelativeId = (id: string) =>
+    id.startsWith('.') ? '/' + id : id;
```

Comment:
This is really hacking into Vite 😄 I think it's safer to use `path.resolve` here instead so it's an absolute path.

Reply:
My first approach was indeed `path.resolve(config.root, id)` which worked for me on Mac, but I wondered if it breaks in Windows. Is `import 'C:\user\dir\file.css'` valid Javascript?

Reply:
> Is `import 'C:\user\dir\file.css'` valid Javascript?

It won't work in node because `C:` looks like a protocol, but it will work in Vite as it's plugin can handle that. So it should also work in Windows.

Reply:
Ahh, Vite, the beautiful land where dreams come true. Thanks @bluwy!

Will update to this (adding the `resolve` import up top):

```suggestion
  const resolveRelativeId = (id: string) =>
    id.startsWith('.') ? resolve(fileURLToPath(root), id) : id;
```